main/libressl/APKBUILD

```diff
@@ -2,9 +2,9 @@
 # Contributor: Orion <systmkor@gmail.com>
 # Maintainer: Orion <systmkor@gmail.com>
 pkgname=libressl
-pkgver=2.5.3
+pkgver=2.5.4
 _namever=${pkgname}${pkgver%.*}
-pkgrel=0
+pkgrel=1
```

Comment:
Ah, it was already updated and GitHub was weird about showing changes -- nevermind then!